### PR TITLE
DSA 1.2.2.4 Algorithms: Recurrence Relations : Recursive_Tree_Method#485

### LINKED ISSUE #485: DSA 1.2.2.4 Algorithms: Recurrence Relations
## Description
How can you solve recurrences using Recursion Tree Method? Discuss the approach and provide examples.

## Task
- [ ] Documentation
- [ ] Audio
- [ ] Video

## Note
- While requesting the issue to be assigned, please mention your **NAME & BATCH NUMBER**.
- Kindly refer to the [Issue Guidelines](https://github.com/girlscript/winter-of-contributing/blob/DSA/DSA/CONTRIBUTING.md#issues).
- In case you see that the issue has already been assigned, try to request for a different task within the same issue.
- Changes should be made inside the ````DSA/directory```` & ````DSA```` branch.
- Name of the file should be same as the issue.
- This issue is only for 'GWOC' contributors of 'DSA' Domain.

### PR DESCRIPTION
**Pull request by RBiswa787**
<hr>

## Description 📜

Added Documentation for Recursive Tree Methods. 
Fixes #485 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [x] DSA
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by RBiswa787** [comment]
@nimisha-yadav Completed the documentation. Kindly review the PR. 
Thank you

**Comment by nimisha-yadav** [comment]
Please review @sneha-nitdgp @harshsri28

**Comment by nimisha-yadav** [comment]
@RBiswa787 kindly edit your issue number and remove the space between # and digits then your issue will be linked.

File: DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.4_Recursive_Tree_Method/1.2.2.4_Recursive_Tree_Method.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+![](Images/generic_tree.png)
+
+1. The Recursion Tree for a given recurrence relation is drawn. At every level, the parent problem is divided into multiple subproblems, until cost of the subproblem becomes 1.
+
+2. We find the total number of leaf nodes and the total cost of the last level.
+
+3. We find a relation to determine the total cost of the kth level. Total cost of level 1 to k-1 is calculated.
+
+4. Total Cost of algorithm = Total cost of leaf nodes + Total cost of internal nodes.
```
Comment (harshsri28) [inline DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.4_Recursive_Tree_Method/1.2.2.4_Recursive_Tree_Method.md:19]: Highlight this statement
eg->  `Total cost of leaf nodes + Total cost of internal nodes.`

**Review by harshsri28 — CHANGES_REQUESTED** [review]

File: DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.4_Recursive_Tree_Method/1.2.2.4_Recursive_Tree_Method.md (hunk 1 above)
Comment (RBiswa787) [reply DSA/1.2 Analysis of Algorithms/2. Recurrence Relations/1.2.2.4_Recursive_Tree_Method/1.2.2.4_Recursive_Tree_Method.md]: @harshsri28 Updated and highlighted steps under generic process.

**Review by harshsri28 — APPROVED** [review]
LGTM

**Review by nimisha-yadav — APPROVED** [review]
Good work. Congrats on getting your PR merged.

**Pull request merged**